.travis.yml

```diff
@@ -12,4 +12,4 @@ script:
 - cd service_management/azure && bundle install && bundle exec rake test:unit
 - if [ "$TRAVIS_ASM_INTEG_RECORDED" == "true" ] ; then bundle exec rake test:recorded ; fi
 - cd ../../resource_management && bundle install
-- if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ] ; then rake test:integration ; fi
\ No newline at end of file
+- if [ "$TRAVIS_ASM_INTEG_RECORDED" == "true" ] ; then rake arm:recorded ; fi
```

Comment:
Since the variable is called ASM, maybe we want to have a different one for ARM; or if we choose to have the same we could remove ASM from the variable name.